doc/install/index.rst

```diff
@@ -1,62 +1,13 @@
-.. include:: ../links.inc
+.. _installers:
 
-.. _quick-start:
-
-Quick start
-===========
-
-MNE-Python requires Python version |min_python_version| or higher. If you've
-never worked with Python before, skip ahead to the last paragraph of this page.
-For users already familiar with Python:
-
-- If you only need MNE-Python's computational functions, only hard dependencies
-  will be included when running:
-
-  .. code-block:: console
-
-      $ pip install mne
-
-- If you plan to use MNE-Python's functions that use HDF5-based I/O (e.g.,
-  :func:`mne.io.read_raw_eeglab`, :meth:`mne.SourceMorph.save`, etc.),
-  you should run:
-
-  .. code-block:: console
-
-      $ pip install mne[hdf5]
-
-  This will pull in additional dependencies pymatreader_, h5io_, and h5py_.
-
-- If you need MNE-Python's 3D rendering capabilities (e.g., plotting estimated
-  source activity on a cortical surface) it is a good idea to install
-  MNE-Python into its own virtual environment. To do this with
-  `conda <anaconda_>`_:
-
-  .. code-block:: console
-
-      $ conda create --name=mne --channel=conda-forge mne
-      $ #                   ↑↑↑                       ↑↑↑
-      $ #             environment name            package name
-
-  This will create a new ``conda`` environment called ``mne`` and install all
-  dependencies into it. If you need to convert structural MRI scans into models
-  of the scalp, inner/outer skull, and cortical surfaces you also need
-  :doc:`FreeSurfer <freesurfer>`.
-
-For users unfamiliar with Python, the :ref:`standard_instructions` page has
-detailed instructions for different
-operating systems, and there are instructions for :ref:`install-python`
-if you don't already have it. The :ref:`advanced_setup` page has additional
-tips and tricks for special situations (servers, notebooks, CUDA, installing
-the development version, etc). The :ref:`contributing` has additional
-installation instructions for (future) contributors to MNE-Python (e.g, extra
-dependencies for running our tests and building our docs).
+.. include:: installers.rst
 
 .. toctree::
     :hidden:
 
-    pre_install
-    install_python
-    mne_python
+    installers
+    manual_install
+    manual_install_advanced
     updating
     freesurfer
-    advanced
+    mne_tools_suite
```

Comment:
I would leave the mne_tools_suite as the first link. It's weird to have the overview at the end

Reply:
My feeling is actually that it doesn't belong into the Install section at all. I'd love to get rid of it or move it elsewhere. I always found it very confusing that the docs talk about a "suite" in the installation section, but then the rest of the docs doesn't touch upon this (not **do** we have installation instructions for the packages of the suite!)